src/core/Akka/Actor/Settings.cs

```diff
@@ -80,7 +80,7 @@ public Settings(ActorSystem system, Config config, ActorSystemSetup setup)
 
             var providerSelectionSetup = Setup.Get<BootstrapSetup>()
                 .FlatSelect(_ => _.ActorRefProvider)
-                .Select(_ => _.Identifier)
+                .Select(_ => _.Fqn)
```

Comment:
Fixed a bug here - caused us not to load any `ProviderSelection.Custom` instances correctly